templates/replication/replication_content.xml.erb

```diff
@@ -12,9 +12,12 @@
         protocolHTTPExpired="<%= @protocol_http_expired %>"
         retryDelay="<%= @retry_delay %>"
         reverseReplication="{Boolean}<%= @reverse_replication %>"
+<% if @reverse_replication == 'true' -%>
```

Comment:
We might want to extract this out into replication_queues.pp.  protocolHTTPmethod would get set to 'UNSET' if not set, but reverse replication is true